code/script.js

```diff
@@ -1,47 +1,252 @@
+const card = () => {
+  questionNumber++
+  
+  botReply(`Would you like to add a greeting card with your flowers?`)
+  
+  inputWrapper.innerHTML=
+  `<button id="cardYesBtn">Yes</button>
+  <button id="cardNoBtn">No</button>`
+  
+  document.getElementById('cardYesBtn').addEventListener('click', () => {
+  cardYes="Yes"
+  
+  nextQuestion('Yes')
+  })
+
+  document.getElementById('cardNoBtn').addEventListener('click', () => {
+    questionNumber=5;
+    nextQuestion('No')
+  }) 
+}
+
+// Question number 5
+const cardGreeting = () => {
+  questionNumber++
+  
+  botReply (`Type your greeting here, for example ''Happy Birthday Lisa, Love Tom''`)
+  
+  inputWrapper.innerHTML=
+  `<form id="formCardGreeting">
+    <input id="inputField" type="text"/>
+    <button id="sendBtn" class="send-btn" type="submit">Send</button>
+  </form>`
+  
+  document.getElementById('formCardGreeting').addEventListener('submit', (event) => {
+    event.preventDefault()
+    cardText= inputField.value
+    nextQuestion(inputField.value)
+  })
+}
+
+// Question number 6
+const deliveryTime = () => {
+  questionNumber++
+
+  botReply (`What time would you like your delivery?`)
+  
+  inputWrapper.innerHTML=
+  `<select id="deliveryTime">
+    <option value="" selected disabled>👇 Select a delivery time...</option>
+    <option value="10AM-12PM">10AM-12PM</option>
+    <option value="12PM-2PM">12PM-2PM</option>
+    <option value="2PM-4PM">2PM-4PM</option>
+    <option value="4PM-6PM">4PM-6PM</option>
+  </select>`
+  
+  let select = document.getElementById('deliveryTime')
+  
+  select.addEventListener('change', () => {
+   delTime= select.value
+   nextQuestion(select.value)
+  })
+}
+
+//Question number 7 
+const finalMessage = () => {
+  if (cardYes === "Yes"){
+    botReply(`Thank you for your order. We will deliver ${flowers} with 
+    a card reading "${cardText}" to the address: ${address} between ${delTime}. 
+    Please press "confirm" to continue with the order or "restart" to make changes`)
+  }
+
+  else {
+    botReply(`Thank you for your order. We will deliver ${flowers} to the address: ${address} between ${delTime}. 
+    Please press "confirm" to continue with the order or "restart" to make changes`)
+  }
+
+  inputWrapper.innerHTML=
+  `<button id="confirmBtn">Confirm</button>
+  <button id="restartBtn">Restart</button>`
+
+  document.getElementById('confirmBtn').addEventListener('click', () => {
+   document.getElementById('chatdiv').style.display="none"
+   document.getElementById('confirmpage').style.display="flex"
+    
+  })
+  document.getElementById('restartBtn').addEventListener('click', () => {
+    questionNumber=1
+    
+    flowers = "" //resetting variable to empty for new choices
+    address = "" //resetting variable to empty for new choices
+    cardText = "" //resetting variable to empty for new choices
+    delTime = "" //resetting variable to empty for new choices 
+    cardYes = "" //resetting variable to empty for new choices
+    
```

Comment:
`    document.querySelector(".chat").innerHTML = "";  
`

If you replace this code, it will clean everything. That worked for me, perhaps you can test it.